lib/buildpackutil.py

```diff
@@ -278,25 +281,34 @@ def fix_mono_config_and_get_env(dot_local_location, mono_lib_dir):
         's|/app/vendor/mono/lib/libgdiplus.so|%s|g' % os.path.join(
             mono_lib_dir, 'libgdiplus.so'
         ),
-        os.path.join(_get_mono_path(dot_local_location), 'etc/mono/config'),
+        os.path.join(_get_mono_path(dot_local_location, mendix_version), 'etc/mono/config'),
     ])
     subprocess.check_call([
         'sed',
         '-i',
         's|/usr/lib/libMonoPosixHelper.so|%s|g' % os.path.join(
             mono_lib_dir, 'libMonoPosixHelper.so'
         ),
-        os.path.join(_get_mono_path(dot_local_location), 'etc/mono/config'),
+        os.path.join(_get_mono_path(dot_local_location, mendix_version), 'etc/mono/config'),
     ])
     return env
 
 
-def _get_mono_path(dot_local_location):
+def _get_mono_version(mendix_version):
+    if LooseVersion(mendix_version) < LooseVersion('6.8'):
+        target = 'mono-3.10.0'
+    else:
+        target = 'mono-4.4.1.0'
+    logging.info('Targeting Mono Runtime: ' + target)
+    return target
+
+
+def _get_mono_path(dot_local_location, mendix_version):
+    mono_version = _get_mono_version(mendix_version)
     return get_existing_directory_or_raise([
-        os.path.join(dot_local_location, 'mono'),
-        '/opt/mono-3.10.0',
-        '/usr/local/share/mono-3.10.0',
```

Comment:
AFAIK this line should stay here for backwards compatibility, since we specifically create a symlink while building the rootfs, see commit `0e8e5db423c9e57a4c641646fc51b6a47a701340` in Gitlab `cf-tools-bash`, file `mxlinuxfs2/mxrootfs.sh`.